### PR TITLE
Feature/insert color class

### PR DESCRIPTION
**Pull request by sjumba**
I have modified the existing end point, where I insert a portion of code into svg and convert back it to dataUrl before its sent out.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
 import { svgo } from "../SvgBackend/src/svgo.config";
 import SVGO from "svgo";
-import cors from 'cors';
+import cors from "cors";
 
 const app = express();
 const port = process.env.PORT || 3500;
 const svgoObject = new SVGO(svgo);
+const svgCodeToBeInserted =
+  "<defs><style>.colour-class{fill:#FFFFFF;}</style></defs>";
```
Comment (MpiloMshengu) [inline server.js]: create constant file

File: server.js [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(cors());
 
-app.post('/optimizeSvg',function(req,res){
-    svgoObject.optimize(dataUriToBuffer(req.body.dataUrl).toString())
+app.post("/optimizeSvg", function(req, res) {
```
Comment (MpiloMshengu) [inline server.js]: add it to constant file

File: server.js (hunk 1 above)
Comment (sjumba) [reply server.js]: done

File: server.js (hunk 2 above)
Comment (sjumba) [reply server.js]: done

**Review by MpiloMshengu — APPROVED** [review]

**Review by Andileh — APPROVED** [review]

File: server.js [hunk 3]
```diff
@@ -1,25 +1,42 @@
...
 app.use(cors());
 
-app.post('/optimizeSvg',function(req,res){
-    svgoObject.optimize(dataUriToBuffer(req.body.dataUrl).toString())
+app.post(OPTIMIZE_SVG, function(req, res) {
+  svgoObject
+    .optimize(dataUriToBuffer(req.body.dataUrl).toString())
     .then(result => {
-        res.send(JSON.stringify({urlData:svgToDataURL(result.data)}));
+      var newDataWithColorClass = " ";
```
Comment (CastelloDev) [inline server.js]: make this let please

File: server.js [hunk 4]
```diff
@@ -1,25 +1,42 @@
...
+      if (result.data.toString().search(SVG_CODE_TO_BE_INSERTED) < 0) {
+        const PositionToInsert = result.data.toString().indexOf(">") + 1;
```
Comment (CastelloDev) [inline server.js]: camel case please

**Review by CastelloDev — CHANGES_REQUESTED** [review]

File: server.js [hunk 5]
```diff
@@ -1,25 +1,42 @@
...
+        newDataWithColorClass = [
+          result.data.toString().slice(0, PositionToInsert),
+          SVG_CODE_TO_BE_INSERTED,
+          result.data
+            .toString()
+            .slice(PositionToInsert, result.data.toString().length)
+        ].join();
+      } else {
+        newDataWithColorClass = result.data;
```
Comment (CastelloDev) [inline server.js]: Can you create a utils folder with a file called svgHelper and add this logic into a function called declareColourClass then import that function into this file and call it in this method

File: server.js (hunk 3 above)
Comment (sjumba) [reply server.js]: Done.

File: server.js (hunk 4 above)
Comment (sjumba) [reply server.js]: attended.

File: server.js (hunk 5 above)
Comment (sjumba) [reply server.js]: Done deal.

**Review by CastelloDev — APPROVED** [review]

**Pull request merged**